runtime/drivers/admin/admin.go

```diff
@@ -143,7 +143,9 @@ func (h *Handle) Ping(ctx context.Context) error {
 	// check connectivity with admin service
 	_, err := h.admin.Ping(ctx, &adminv1.PingRequest{})
 
-	_ = h.repoMu.RLock(ctx)
+	if lockErr := h.repoMu.RLock(ctx); lockErr != nil {
+		return lockErr
+	}
```

Comment:
```suggestion
	if err := h.repoMu.RLock(ctx); err != nil {
		return err
	}
```

Reply:
There is already a `err` declared above this line. I just feel better to use different names to avoid introducing bugs.

Reply:
Hadn't seen, makes sense